molecule/default/molecule.yml

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: MIT
+---
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint:
+  name: yamllint
+  options:
+    config-file: .yamllint.yml
+platforms:
+  - name: centos-6
+    image: docker.io/linuxsystemroles/centos-6
+    privileged: true
```

Comment:
is this role supported on centos-6?  If not, you can remove this

we don't yet have a container image for centos-8 support

Reply:
I don't think we'd need centos-6.

Reply:
34ed562fda1139e3b90806c54908fab2d1ac0119

Reply:
34ed562fda1139e3b90806c54908fab2d1ac0119